### PR TITLE
Put the `lang-team` repository under automation

### PR DESCRIPTION
**Pull request by Kobzol**

### CODE REVIEW

File: repos/rust-lang/lang-team.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "lang-team"
+description = "Home of the Rust lang team"
+bots = ["rustbot"]
```
Comment (rylev) [inline repos/rust-lang/lang-team.toml:4]: It seems they also use RFC bot.

File: repos/rust-lang/lang-team.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+[access.teams]
+lang = "maintain"
```
Comment (rylev) [inline repos/rust-lang/lang-team.toml:7]: Release also has write permissions currently cc @Mark-Simulacrum

Comment (nikomatsakis) [reply repos/rust-lang/lang-team.toml:7]: let's also add the various lang subteams:

* ops
* docs
* types

**Review by nikomatsakis — CHANGES_REQUESTED** [review]
thank you!

**Comment by Kobzol** [comment]
> let's also add the various lang subteams:

All with the `maintain` permission? Or just read/write?

File: repos/rust-lang/lang-team.toml (hunk 2 above)
Comment (Kobzol) [reply repos/rust-lang/lang-team.toml:7]: All with the maintain permission? Or just read/write?

**Comment by Kobzol** [comment]
I added the RFCbot and more teams, with write permissions. Let me know if I should change the permissions.

File: repos/rust-lang/lang-team.toml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+name = "lang-team"
+description = "Home of the Rust lang team"
+bots = ["rustbot", "rfcbot"]
+
+[access.teams]
+lang = "maintain"
+lang-ops = "write"
+lang-docs = "write"
+types = "write"
+release = "write"
```
Comment (nikomatsakis) [inline repos/rust-lang/lang-team.toml:11]:
```suggestion
lang-ops = "maintain"
lang-docs = "maintain
types = "maintain"
release = "maintain"
```

looking at github, I see that these things are maintain...

image

I think at *least* ops needs those, so why not the others too.

**Review by nikomatsakis — CHANGES_REQUESTED** [review]

File: repos/rust-lang/lang-team.toml (hunk 3 above)
Comment (Kobzol) [reply repos/rust-lang/lang-team.toml]: Ok, changed to `maintain` :)

**Review by nikomatsakis — APPROVED** [review]
Looks great to me!

**Review by Mark-Simulacrum — APPROVED** [review]

**Pull request merged**